File: Java/Product.java
// kelas yang digunakan untuk mengimplementasikan product
class Product {
    // private atribute dari kelas product
    private int price;
    private int idProduct;

    // konstrutor
    Product() {
    }

    // mengeset nilai atribut price
    public void setPrice(int price) {
        this.price = price;
    }

    // mengembalikan nilai atribut price
    public int getPrice() {
        return this.price;
    }

    // mengeset nilai atribut idProduct
    public void setIdProduct(int idProduct) {
        this.idProduct = idProduct;
    }

    // mengembalikan nilai atribut idProduct
    public int getIdProduct() {
        return this.idProduct;
    }

    // menampilkan atribut product
    public void printProduct() {
        System.out.println("Id Product : " + getIdProduct());
        System.out.println("Price : " + getPrice());
    }
}

File: PHP/Product.php
<?php
class Product
# kelas yang digunakan untuk mengimplementasikan product
{
    # private atribute dari kelas product
    private $price = 0;
    private $idProduct = 0;

    # konstrutor
    public function __construct()
    {
    }

    # mengeset nilai atribut price
    public function setPrice($price)
    {
        $this->price = $price;
    }

    # mengembalikan nilai atribut price
    public function getPrice()
    {
        return $this->price;
    }

    # mengeset nilai atribut idProduct
    public function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }

    # mengembalikan nilai atribut idProduct
    public function getIdProduct()
    {
        return $this->idProduct;
    }

    # menampilkan atribut product
    public function printProduct()
    {
        echo "Id Product: " . $this->getIdProduct() . "<br/>";
        echo "Price: " . $this->getPrice() . "<br/>";
    }
}
